src/schema/v2/homeView/sections/QuickLinks.ts

```diff
@@ -16,36 +16,55 @@ export const QuickLinks: HomeViewSection = {
 
 const QUICK_LINKS: Array<NavigationPill> = [
   {
-    title: "Follows",
-    href: "/favorites",
-    ownerType: OwnerType.follows,
-    icon: "FollowArtistIcon",
+    title: "Saves",
+    href: "/favorites/saves",
+    ownerType: OwnerType.saves,
+    icon: "HeartStrokeIcon",
   },
   {
     title: "Auctions",
     href: "/auctions",
     ownerType: OwnerType.auctions,
-    icon: "AuctionIcon",
+    icon: "GavelIcon",
   },
   {
-    title: "Saves",
-    href: "/favorites/saves",
-    ownerType: OwnerType.saves,
-    icon: "HeartIcon",
+    title: "New This Week",
+    href: "/collection/new-this-week",
+    ownerType: OwnerType.collection, // TODO: collection id?
+    icon: undefined,
   },
   {
-    title: "Art under $1000",
-    href: "/collect?price_range=%2A-1000",
-    ownerType: OwnerType.collect,
+    title: "Editorial",
+    href: "/articles",
+    ownerType: OwnerType.articles,
+    icon: "PublicationIcon",
   },
   {
-    title: "Price Database",
-    href: "/price-database",
-    ownerType: OwnerType.priceDatabase,
+    title: "Statement Pieces",
+    href: "/collection/statement-pieces",
+    ownerType: OwnerType.collection, // TODO: collection id?
+    icon: undefined,
   },
+
+  // TODO: enable once navigation is sorted out
+  // {
+  //   title: "Medium",
+  //   href: "/collections-by-category/Medium",
```

Comment:
This one is tough! I tried to build the correct href, but failed because of the [unusual way](https://github.com/search?q=repo%3Aartsy%2Feigen%20collections-by-category&type=code) in which we are routing from the home view to those screens. This will need some attention, but we can also merge here and follow up.